apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+
+1. Navigate to the .env file within your solution under src\sxastarter, and copy it.  
+
+2. Name it .env.local. We do this so that the file is ignored by git by default, so that you do not pass valuable configuration secrets into any repositories.  
+ 
+You can connect using the delivery endpoint of Edge or to the preview endpoint of your CM instance.   
+![XM Cloud Architecture - Connect to Preview Endpoint](https://sitecorecontenthub.stylelabs.cloud/api/public/content/e703be66170a49268beb6564c4c34df7?v=5184aace) 
+
+Any app showing content to the public must connect to the Edge Delivery Endpoint that is scaled and geographically distributed to serve the right performance. Only published content will be available here. For development purposes, you will connect to the preview endpoint as you do not need to care about publishing at this point, so created content is instantly available. This will simplify the development workflow.    
+ 
+
+Now, you need to set the API key within .env.local. The API key should be already available within your XM Cloud instance.  
+ 
+3. Now you need to set the Context ID in the .env.local file. Therefore  
+ * Go to the XM Cloud deploy app 
+ * Click on the name of the project you created e.g. XM Cloud Tutorial Series 
+ * Copy the Context ID (Preview) 	 
+![Get Context ID from XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8ecc65168a504cc188ee84cd8951244b?v=a04033a5) 
+4. Add the Context ID to the `.env.local` file 
+![Copy Context ID into .env file](https://sitecorecontenthub.stylelabs.cloud/api/public/content/e29d1bb0c6a448efb63dce32af047d49?v=2edbe4d1) 
+5. Add the Sitename to the `.env.local` which should be “Company Dev”
+![Set Site Name in .env file](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8fbd4505019948f08bfba8c8be4d4043?v=79a1d301) 
+6. In your terminal navigate to src/sxastarter (your app folder) and run `npm install`
+![In Command Line Interface - Run npm install](https://sitecorecontenthub.stylelabs.cloud/api/public/content/38c87be1d50c4b40abab26556102c4f6?v=474160b6) 
+7. Now run npm run start:connected
+![Start the app locally](https://sitecorecontenthub.stylelabs.cloud/api/public/content/ad26063be71c4ae6b9e62e2579948db3?v=1a82e9cc) 
+8. Call your app in the browser http://localhost:3000
+![Check Website running locally](https://sitecorecontenthub.stylelabs.cloud/api/public/content/358a02664690465289d70fe4c9280eae?v=7c0165d9) 
+ 
+#### Verifying the connection  
+
+ 
+
+Let’s change some content to verify you are in fact connected against the preview endpoint. 
+1. Navigate back to your site in XM Cloud. 
+![XM Cloud Dashboard - Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8da9bc8363914bbda2b8b4a67e0cd31e?v=00d2e820) 
+2. Click on the tile that says Company Dev. This will open Pages new WYSIWYG Page Editing experience. When opening for the first time, this might take a while.  
```

Comment:
* "Company Dev" should be emphasized (suggest `Company Dev` formatting)
* `Page Editing` should be `page editing`
* I suggest `take a moment` rather than `take a while` to make it less negative

Reply:
Resolved in latest commit